code/datums/components/contraband.dm

```diff
@@ -70,8 +70,8 @@ TYPEINFO(/datum/component/contraband)
 	src.contraband_logic(owner, user, slot_mult)
 
 /datum/component/contraband/proc/removed(obj/item/owner, mob/user)
-	REMOVE_ATOM_PROPERTY(user, PROP_MOVABLE_VISIBLE_GUNS, src)
-	REMOVE_ATOM_PROPERTY(user, PROP_MOVABLE_VISIBLE_CONTRABAND, src)
+	REMOVE_ATOM_PROPERTY(user?, PROP_MOVABLE_VISIBLE_GUNS, src)
+	REMOVE_ATOM_PROPERTY(user?, PROP_MOVABLE_VISIBLE_CONTRABAND, src)
```

Comment:
This isn't how DM works, if you want this code not to run when user is null then you will need to use an if statement.